docs/scos/user/features/202204.0/product-barcode-feature-overview.md

```diff
@@ -53,15 +61,10 @@ Barcodes are often overlooked as a way to cut costs and save time. A valuable an
 The Barcode Generator can be used for any kind of entity, and by default, Spryker provides a solution for products.
 
 **What is a barcode?**
```

Comment:
delete. Also, please delete the "What is the barcode" heading in the very beginning of this document, after the first sentence. The new "What is the barcode" section does not make sense